logcat/src/jsTest/kotlin/logcat/JsLogcatTest.kt

```diff
@@ -0,0 +1,40 @@
+package logcat
+
+import logcat.LogPriority.INFO
+import kotlin.test.AfterTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class JsLogcatTest {
+
+  @AfterTest
+  fun tearDown() {
+    LogcatLogger.uninstall()
+  }
+
+  @Test fun logcat_passes_priority_to_isLoggable_check() {
+    var receivedPriority: LogPriority? = null
+    TestLogcatLogger(isLoggable = { receivedPriority = it; true })
+      .apply { LogcatLogger.install(this); latestLog = null }
+
+    logcat(INFO) { "Hi" }
+
+    assertEquals(INFO, receivedPriority)
+  }
+
+  @Test fun Throwable_asLogMessage_has_stacktrace_logged() {
+    val logger = TestLogcatLogger().apply { LogcatLogger.install(this); latestLog = null }
+    val exception = RuntimeException("damn")
+
+    logcat { exception.asLog() }
+
+    val stackTraceStr = """
+      |RuntimeException: damn
+      |    at JsLogcatTest.Throwable_asLogMessage_has_stacktrace_logged
```

Comment:
Is this dependent on the JS engine?

Reply:
Appears to be part of the Kotlin stdlib for JS, but does use the `asDynamic().stack` which could be JS engine specific.  FWIW, the test did pass when I added nodejs in addition to browser as targets in the build.gradle.  I'll have to find time to dig in more to find more of an answer.
 https://github.com/JetBrains/kotlin/tree/d8b0dbe71bbeb84ba0fbdac87801c8c6f4eed4c9/libraries/stdlib/js/src/kotlin/throwableExtensions.kt#L18 uses ExceptionTraceBuilder.dumpFullTrace/dumpSelfTrace.

Reply:
Looks like the `.stack` is dependent on JS engine:

v8 - https://v8.dev/docs/stack-trace-api, https://nodejs.org/api/errors.html#errorstack
mdn - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/stack#examples